documents/affiliated_package_review_guidelines.md

```diff
@@ -1,9 +1,22 @@
 # Reviewing affiliated packages
 
-This document describes the set of review criteria for packages applying to
-become Astropy affiliated.
+If you are reading this because you have accepted to review an affiliated
+package submission, thank you for taking the time to do this!
```

Comment:
Maybe still include something like "this document describes ..."?  Right now the text provides no obvious context about what this document *is*.